src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  errorMessage: string;
+}
+
+export const ErrorNotification: React.FC<Props> = ({ errorMessage }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: !errorMessage },
+      )}
+    >
+      <button data-cy="HideErrorButton" type="button" className="delete" />
+      {errorMessage && <div>{errorMessage}</div>}
```

Comment:
You don't need to use logical operator here
Because your notification will be hidden and not displayed if there is no errorMessage